EURO2021 abstract (#3)
11michalis11/Talks: EURO2021/Abstract.tex

\documentclass{article}

> \title{A 3-player game theoretic model of a choice between two queueing systems with strategic managerial decision making}

Reviewer: Nice.

\author{Michalis Panayides}

\begin{document}

\maketitle
\begin{abstract}

    The main focus of this study is the construction of a 3-player game 
    theoretic model between two queueing systems and a service that distributes 
    individuals to them. 
    The resultant model will then be used to explore dynamics between all 
    players.

    The first aspect of this work is the development of a queueing system

>     with two waiting spaces and two types of individuals. 

Reviewer: ```suggestion
    with two consecutive waiting spaces. The strategic managerial behaviour corresponds to how individuals use these waiting spaces.
```

    Two modelling techniques were used, discrete event 
    simulation and Markov chains. 
    The state probabilities of the Markov chain system have been used to extract
    the performance measures of the queueing model (e.g. mean time in each 

>     waiting room, mean number of individuals in each room, etc.).

Reviewer: ```suggestion
    waiting room, mean number of individuals in each room, etc.).
    
```

>     Given two such systems, the scenario explored is
>     how to distribute individuals among the two in order to optimise certain
>     performance measures. 

Reviewer: ```suggestion

```

    A 3-player game theoretic model is proposed between the two
    queueing systems and the service that distributes individuals to them. 
    In particular this can be seen as a 2-player normal-form game where the 
    utilities are determined by a third player with its own strategies and 
    objectives. 
    A backwards induction technique is used to get the utilities of the 
    normal-form game between the two queueing systems.

    This particular system can be applied in a healthcare scenario where it 

>     could capture the emergent behaviour between the Emergency Medical Service 

Reviewer: ```suggestion
    captures the emergent behaviour between the Emergency Medical Service 
```

    (EMS) and the Emergency Department (ED). 

>     The model will be used to form a 3-player game between the EDs of two 
>     neighbouring hospitals and the EMS.

Reviewer: ```suggestion
    This will be used to investigate the impact of target measures on patient welfare.
```

\documentclass{article}

> \title{A game theoretic model of the behavioural gaming that takes place at the 
> EMS - ED interface}
> \author{Michalis Panayides}

Reviewer: ```suggestion
\title{A 3 player game theoretic model of a choice between two queueing systems with strategic managerial decision making}
\author{Michalis Panayides}
```

The healthcare part is not the "main" attraction for the EURO talk.

\begin{document}

\maketitle
\begin{abstract}
    

>     Operational Research (OR) techniques provide a toolkit of mathematical 
>     modelling approaches that are routinely used for problem solving in many
>     sectors. Healthcare is considered to be one of such sectors where, in recent
>     years, OR has found several applications.

Reviewer: ```suggestion
```

Not necessary for the audience.
